scout/server/templates/utils.html

```diff
@@ -33,6 +33,35 @@
   </table>
 {% endmacro %}
 
+
+{% macro variant_related_comments_table(variant, case) %}
+      <table role="presentation" id="panel-table" class="table table-sm" style="background-color: transparent">
+        <thead>
+          <tr>
+            <td colspan=2><strong>Variant-related comments</strong></td>
+          </tr>
+        </thead>
+        <tbody>
+          {% for comment in variant.comments %}
+            <tr>
+              <td style="width:50%;">{{comment.created_at.strftime('%Y-%m-%d')}} {{comment.user_name}}
+                {% if comment.level == 'global' %}
+                  <span class='badge bg-info'>GLOBAL</span>
+                {% endif %}
+                {% if comment.created_at < case.updated_at %}
+                  <span class='badge bg-warning'>OLD</span>
+                {% endif %}
+                {% if comment.created_at < comment.updated_at %}
+                  <span class='badge bg-secondary'>edited</span>
```

Comment:
The text inside is not super readable: 

<img width="257" alt="image" src="https://user-images.githubusercontent.com/28093618/222115597-b265c8c3-b529-4c85-8a8f-41b1e1eb6733.png">

Perhaps it would be better white?

Reply:
Agreed, let's try!

Reply:
Yep, better! Changed also the other macro. It is eh actually very similar. But enough now, we will have a good opportunity merging it when moving reports to bs5 - or bs6. 😄